lib/mix/tasks/ops.handle_commit.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Mix.Tasks.Ops.HandleCommit do
+  use Mix.Task
+
+  @shortdoc "Handle commit message"
+
+  @moduledoc """
+  Parse last commit message. If on last row we detect commands, we execute them ('-f' - this flag mean fast)
+
+  Examples:
+
+  build
+  uat/staging
+  uat~-f
+  """
+
+  @doc false
+  def run(_args) do
+    commands = Ops.Utils.Git.lookup_commands_from_commit_message()
```

Comment:
is it still inside the utils?
i think it's a better to extract to separated command